src/Combinatorics/EnumerativeCombinatorics/combinations.jl

```diff
@@ -18,7 +18,7 @@ julia> collect(C)
  [2, 3, 4]
 ```
 """
-combinations(n::Int, k::Int) = Combinations(1:n, k)
+combinations(n::Int, k::Int) = Combinations(range(; length=n), k)
```

Comment:
Maybe like this? That would make it easier to "find all places related to the `OneTo` optimization" via grepping.
```suggestion
combinations(n::Int, k::Int) = Combinations(Base.OneTo(n), k)
```

Reply:
Ah and this is also necessary for this to work in Julia 1.6